basepage/templates/basepage/contact.html

```diff
@@ -7,7 +7,7 @@ <h1>
     Message sent
 {%else%}
 
-<form method="POST" action="/contact/">
+<form method="POST" action="/contact/" >
```

Comment:
This is the only change in this file and if you would have reverted it the files would not have been included in the commit.